src/Sulu/Bundle/ContentBundle/Tests/Functional/Command/CleanupHistoryCommandTest.php

```diff
@@ -47,16 +56,39 @@ public function setUp()
         $this->tester = new CommandTester($cleanupCommand);
     }
 
+    private function createContentNode(NodeInterface $parent, $webspaceKey, $title, $url, $locale)
+    {
+        $node = $parent->addNode($title);
+        $node->setProperty('i18n:' . $locale . '-template', 'default');
+        $node->setProperty('i18n:' . $locale . '-creator', 1);
+        $node->setProperty('i18n:' . $locale . '-created', new \DateTime());
+        $node->setProperty('i18n:' . $locale . '-changer', 1);
+        $node->setProperty('i18n:' . $locale . '-changed', new \DateTime());
+        $node->setProperty('i18n:' . $locale . '-title', $title);
+        $node->setProperty('i18n:' . $locale . '-state', WorkflowStage::PUBLISHED);
+        $node->setProperty('i18n:' . $locale . '-published', new \DateTime());
+        $node->setProperty('i18n:' . $locale . '-url', $url);
+        $node->addMixin('sulu:page');
```

Comment:
Also its not a good test, as this is not actually how content is created, and the process may change in the future whilst this test will not.